front/src/applications/rollingStockEditor/components/RollingStockEditorForm.tsx

```diff
@@ -420,25 +385,72 @@ const RollingStockEditorForm = ({
     );
   };
 
-  // TODO: use InputSNCF and InputGroupSNCF
+  useEffect(() => {
+    setIsValid(true);
+    RollingStockschemaProperties.forEach((property) => {
+      if (
+        (property.title === Parameter[property.title as Parameter] &&
```

Comment:
```suggestion
        (RollingStockParameters.includes(property.title) &&
```

Reply:
this test is only for parameters, no need of this for metadatas